rholang/src/main/scala/coop/rchain/rholang/interpreter/PrettyPrinter.scala

```diff
@@ -73,7 +73,7 @@ case class PrettyPrinter(freeShift: Int, boundShift: Int) {
 
   def buildString(v: Var): String =
     v.varInstance match {
-      case FreeVar(level) => s"x${freeShift + level}"
+      case FreeVar(level) => s"z${freeShift + level}"
       case BoundVar(level) => s"x${boundShift - level - 1}"
```

Comment:
Nit: I would be happier if this read BoundVar(index) . . .
Feel free to fix post-submit.